pkg/registry-disk/registry-disk.go

```diff
@@ -36,7 +36,7 @@ const filePrefix = "disk-image"
 
 var registryDiskOwner = "qemu"
 
-var mountBaseDir = "/var/run/libvirt/kubevirt-disk-dir"
+var mountBaseDir = "/var/run/kubevirt-ephemeral-disks/registry-disk-data"
```

Comment:
Can we create constants for `ephemeral-disk` and `registry-disk`? Is `pkg/ephemeral-disk` also considered in migrations? Anything special neede for cloud-init, config-map and secret disks? I don't think so, just want to be sure.

Maybe we can in a follow-up PR rearrange the package structure into  something like:

```
pkg/disks/
pkg/disks/constants.go
pkg/disks/registry-disk/
pkg/disks/ephemeral-disk/
...
```

Reply:
yeah, i can make these constants.

These disks get "migrated" in that they are recreated on the host side during the target pod preparation phase. If the secret or config maps change, that could cause some strange conditions after the migration, but I'm not sure if that's something we need to account for yet. For the most part I'm trying to keep storage out of this initial PR.

Reply:
After looking at this more, I think we should hold off on creating constants for a follow up patch. To do this properly, there's quite a bit to untangle. making the constants now without reorganizing the code might make this even more confusing because there's not a reasonable place for the ephemeral disk base path to live.